codestyle/checkstyle.xml

```diff
@@ -171,5 +171,17 @@
       <property name="illegalPattern" value="true"/>
       <property name="message" value="Use java.lang.Primitive.BYTES instead."/>
     </module>
+
+    <module name="Regexp">
```

Comment:
Please add a comment that this regex should be replaced with an IntelliJ inspection when teamcity.jetbrains.com updates to at least IntelliJ 2018.1 (currently it uses 2017.2)

Reply:
Added.